sklearn/neural_network/base.py

```diff
@@ -189,8 +189,40 @@ def log_loss(y_true, y_prob):
     """
     y_prob = np.clip(y_prob, 1e-10, 1 - 1e-10)
 
+    if y_prob.shape[1] == 1:
+        y_prob = np.append(1 - y_prob, y_prob, axis=1)
+
+    if y_true.shape[1] == 1:
+        y_true = np.append(1 - y_true, y_true, axis=1)
+
+    return -np.sum(y_true * np.log(y_prob)) / y_prob.shape[0]
+
+
+def binary_log_loss(y_true, y_prob):
```

Comment:
You mean renaming `binary_log_loss` to `multilabel_log_loss`? This makes sense, it's just that this loss function is actually used in both binary and multilabel case since I made minimal change to `BaseMultilayerPerceptron._initialize()`. I took the 'binary' here from 'binary cross-entropy'.